.travis.yml

```diff
@@ -11,17 +11,19 @@ notifications:
   email:
     recipients:
       - 130s@2000.jukuin.keio.ac.jp
+      - lander@erlerobotics.com
+      - anasarrak@erlerobotics.com
 env:
   global:
     - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
-    - ROS_DISTRO=melodic
+    - ROS_DISTRO=crystal
     - ROS_REPO=ros
     - UPSTREAM_WORKSPACE=moveit.rosinstall
     - TEST_BLACKLIST=moveit_ros_perception  # mesh_filter_test fails due to broken Mesa OpenGL
     - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-function"
     - WARNINGS_OK=false
   matrix:
-    - TEST="clang-format catkin_lint"
```

Comment:
comment this out and make it a TODO 

@rhaschke put a lot of work into this.

Reply:
reverted all travis changes